src/components/Timelog/WeeklySummaries.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const WeeklySummaries = ({ userProfile }) => {
+  return (
+    <div>
+      {userProfile.weeklySummaries[0].summary ? (
+        <div>
+          <h4>This week's summary</h4>
+          <h4>{userProfile.weeklySummaries[0].summary}</h4>
+        </div>
+      ) : (
+        <h4>
+          {userProfile.firstName} {userProfile.lastName} did not submit a summary for this week.
+        </h4>
+      )}
+      {userProfile.weeklySummaries[1].summary ? (
```

Comment:
For a user don't have any summary, the length of `userProfile.weeklySummaries` is 1 and line 16&26 will cause a crash.